lib/routes/withdraw_funds/withdraw_funds_page.dart

```diff
@@ -277,12 +278,25 @@ class WithdrawFundsPageState extends State<WithdrawFundsPage> {
     });
   }
 
-  Future _scanBarcode() async {
+  Future _scanBarcode(AccountModel account) async {
     try {
       FocusScope.of(context).requestFocus(FocusNode());
       String barcode = await QRScanner.scan();
+      String requestAmount;
+      if (barcode.contains("?")) {
```

Comment:
Here I would use as well the `parseBTCAddress` that I suggested to create previously.